examples/production/scc-spire-agent.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: security.openshift.io/v1
```

Comment:
Is this file redundant because of charts/spire/charts/spire-agent/templates/scc-spire-agent.yaml?